apps/full-site-editing/full-site-editing-plugin/full-site-editing/editor/template-validity-override/index.js

```diff
@@ -0,0 +1,25 @@
+/* global fullSiteEditing */
+
+/**
+ * External dependencies
+ */
+import domReady from '@wordpress/dom-ready';
+import { dispatch } from '@wordpress/data';
+
+/**
+ * Forces the template validity.
+ *
+ * This is a work-around for the existing core issue that is showing a template mismatch warning when there is a parent
+ * block with a locked template containing a nested InnerBlocks with an unlocked template.
+ *
+ * @see https://github.com/WordPress/gutenberg/issues/11681
+ */
+function resetTemplateValidity() {
+	if ( 'page' !== fullSiteEditing.editorPostType ) {
+		return;
+	}
+
+	dispatch( 'core/editor' ).setTemplateValidity( true );
```

Comment:
`setTemplateValidity ` has been moved to `core/block-editor` and it's triggering a deprecation warning.

```suggestion
	dispatch( 'core/block-editor' ).setTemplateValidity( true );
```

Reply:
We still need to use `core/editor` in order to support sites not using the latest version: https://github.com/Automattic/wp-calypso/pull/34096

Reply:
Awesome!